src/features/XIT/HEALTH.vue

```diff
@@ -36,7 +36,7 @@ const otherData = computed(() => [
   ['CXOS', cxosStore.all.value?.length],
   ['FXOS', fxosStore.all.value?.length],
   ['Currency', (balancesStore.all.value?.length ?? 0) > 0],
-  ['Last CX Price Update', cxStore.fetched ? `${dayjsEachSecond.value.to(cxStore.age)}` : false],
+  ['Last CX Price Update', cxStore.fetched ? `${dayjsEachMinute.value.to(cxStore.age)}` : false],
```

Comment:
With this you will probably see `in a few seconds` in the `Last CX Price Update` field for a minute before the price update happens. Try reducing the cx price update interval to 3 minutes and see how it goes with `dayjsEachMinute`